.github/workflows/tests.yml

```diff
@@ -64,9 +64,9 @@ jobs:
     - uses: actions/setup-node@v1
       with:
         node-version: 10
-    - uses: microsoft/playwright-github-action@v1
     - run: npm ci
     - run: npm run build
+    - run: node lib/cli/cli install
```

Comment:
Let's try `install ${{ matrix.browser }}` right away?

Reply:
Can't do - we need Chromium for recorder.